app/static/src/app/store/run/mutations.ts

```diff
@@ -122,5 +129,35 @@ export const mutations: MutationTree<RunState> = {
         if (paramSet) {
             paramSet.hidden = !paramSet.hidden;
         }
+    },
+
+    [RunMutation.SaveParameterDisplayName](state: RunState, payload: ParameterSetNames) {
+        const isDuplicateDisplayName = state.parameterSets.find((set: ParameterSet) => {
+            return set.displayName === payload.newDisplayName;
+        });
+        const isSpecialSetName = payload.newDisplayName.match(/^ *set \d+ *$/gi);
+        const paramSet = state.parameterSets.find((set: ParameterSet) => set.name === payload.parameterSetName);
+        if (paramSet) {
+            const isParamSetName = payload.newDisplayName === paramSet.name;
+            const isParamSetDisplayName = payload.newDisplayName === paramSet.displayName;
+            if (isDuplicateDisplayName && !isParamSetName && !isParamSetDisplayName) {
+                paramSet.isDisplayNameError = true;
+                paramSet.displayNameErrorMsg = "Name already exists";
+            } else if (isSpecialSetName && !isParamSetName) {
+                paramSet.isDisplayNameError = true;
+                paramSet.displayNameErrorMsg = `Set [number] combination is reserved for default set names.
+                Please choose another set name or name this set: ${paramSet.name}`;
```

Comment:
```suggestion
                Please choose another set name or name this set back to its original name of '${paramSet.name}'`;
```

Reply:
* Create a parameter set
* Change its name to `example` (or anything)
* Try and change the name to Set 2 (or some other reserved name)
* Change tab and the, and the box goes away, rename operation apparently cancelled (this is fine/good)
* Select Rename, and the edit box shows `example` while the error comes back and says "Set [number] combination..."

Reply:
fixed the issue with error coming back, just needed to turn off error on unmount!